packages/-ember-data/tests/integration/request-state-service-test.ts

```diff
@@ -0,0 +1,229 @@
+import { get } from '@ember/object';
+import { setupTest } from 'ember-qunit';
+import Model from 'ember-data/model';
+import Store from 'ember-data/store';
+import { module, test } from 'qunit';
+import { settled } from '@ember/test-helpers';
+import { identifierCacheFor } from '@ember-data/store/-private';
+import EmberObject from '@ember/object';
+import { attr, hasMany, belongsTo } from '@ember-data/model';
+import { REQUEST_SERVICE } from '@ember-data/canary-features';
+
+class Person extends Model {
+  // TODO fix the typing for naked attrs
+  @attr('string', {})
+  name;
+
+  @attr('string', {})
+  lastName;
+}
+
+if (REQUEST_SERVICE) {
+  module('integration/request-state-service - Request State Service', function(hooks) {
+    setupTest(hooks);
+
+    let store: Store;
+
+    hooks.beforeEach(function() {
+      let { owner } = this;
+      owner.register('model:person', Person);
+      store = owner.lookup('service:store');
+    });
+
+    test('getPendingRequest and getLastRequest', async function(assert) {
```

Comment:
test descriptions should be a bit more declarative of what the scenario being tested is

Reply:
done